### ISSUE TITLE
Have Application Android BLE HUD ?

### ISSUE THREAD

**Issue by huukhoa1412** [alexanderlavrushko/BLE-HUD-navigation-ESP32#5]
Have Application Android BLE HUD ?

**Comment by alexanderlavrushko** [comment]
No, Android application is not available yet. It's still in progress.

The only way to check the connection on Android is to manually send the data:
1. Download BLE utility app, for example [LightBlue for Android](https://play.google.com/store/apps/details?&hl=en&id=com.punchthrough.lightblueexplorer)
2. Run ESP32 BLE HUD
3. Using BLE utility app, scan for devices, connect to a device with name "ESP32 HUD"
4. Find the service with UUID DD3F0AD1-6239-4E1F-81F1-91F6C9F01D86
5. In the service, find the characteristic DD3F0AD3-6239-4E1F-81F1-91F6C9F01D86
6. Write HEX value to that characteristic: 01 32 0A 31 6D

Now ESP32 should show these values: turn right, speed limit 50, text "1m".
More info about format [here](https://github.com/alexanderlavrushko/BLE-HUD-navigation-ESP32#technical-info), see also DataConstants.h

**Comment by modi12jin** [comment]
@alexanderlavrushko  
Hi, I see HUD display in this repo, this might be the answer.

https://github.com/sharandac/My-TTGO-Watch

![ESP32 HUD](https://user-images.githubusercontent.com/40233017/202177493-b48a9985-d314-45ce-b6d2-be033a9986f7.png)

**Comment by modi12jin** [comment]
https://3dprinting.berrycave.tech/2022/10/30/garmin-hud-got-discontinued-but-not-trashed/

https://github.com/skyforcetw/Garminuino

**Comment by modi12jin** [comment]
https://github.com/appleshaman/CarPlayBLE

**Comment by huukhoa1412** [comment]
> https://github.com/appleshaman/CarPlayBLE

thank you
